.github/workflows/openvex.yaml

```diff
@@ -0,0 +1,35 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        required: true
+        description: "GitHub release tag"
+
+permissions: {}
+
+jobs:
+  vexctl:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Comment:
Just from looking over this, I think that we are just using the tag here; so, we could probably use `github.ref_name` rather than having to parse the tag from the ref directly